### ISSUE TITLE
Can't import @polly/core in ember using embroider

### ISSUE THREAD

**Issue by Mciocca** [Netflix/pollyjs#383]
I'm not well versed in embroider and not sure if this is an issue with embroider or polly. Trying to run tests with pollyjs installed fails because it can not find @pollyjs/core. Embroider is the way forward for ember and is needed by some of us in the community for compatibility with 3rd party libraries

```
            Error: Could not find module `@pollyjs/core` imported from `(require)`
                at missingModule (http://localhost:7357/assets/vendor.js:256:11)
                at findModule (http://localhost:7357/assets/vendor.js:267:7)
                at requireModule (http://localhost:7357/assets/vendor.js:33:15)
                at eval (webpack:///../externals/@pollyjs/core.js?:1:18)
                at Object.../externals/@pollyjs/core.js (http://localhost:7357/assets/chunk.1b5c9874276dfbdc23d0.js:165:1)
                at __webpack_require__ (http://localhost:7357/assets/chunk.1b5c9874276dfbdc23d0.js:80:30)
                at eval (webpack:///./tests/acceptance/index-test.js?:5:139)
                at Module../tests/acceptance/index-test.js (http://localhost:7357/assets/chunk.1b5c9874276dfbdc23d0.js:188:1)
                at __webpack_require__ (http://localhost:7357/assets/chunk.1b5c9874276dfbdc23d0.js:80:30)
                at Module.eval [as callback] (webpack:///./assets/test.js?:193:10)
```

This happens in every browser. Using  "@pollyjs/ember": "^5.1.0", yarn 1.22.4

I've reproduced this issue here: https://github.com/Mciocca/polly-embroider-example
     
 I've opened an issue with embroider as well. Please feel free to close this if this is an embroider issue

**Comment by offirgolan** [comment]
Unfortunately I'm not too familiar with emroider as I've been out of the Ember space for some time now. The ember package might need to be updated if this isn't an upstream issue. Are you able to import core on its own without the ember addon?

**Comment by Mciocca** [comment]
@offirgolan  I can get it working if I remove the ember specific package and set it up normally.

**Comment by offirgolan** [comment]
That doesn't seem like it's the issue. Continuing the thread in embroider-build/embroider#767

**Comment by offirgolan** [comment]
Fixed by #415
